examples/gaussian_processes/gaussian_process.ipynb

```diff
@@ -4,18 +4,23 @@
    "cell_type": "markdown",
```

Comment:
<p>There should be a markdown level 2 title right above the watermark code cell. This will make watermark appear on the table of contents (right sidebar) and makes it possible to check the versions a notebook runs on with a single click.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/404/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>